ci/functional/test_main_prep_node.sh

```diff
@@ -63,17 +70,27 @@ set +x
 while IFS= read -r line; do
     ((opa_count++)) || true
 done < <(lspci -mm | grep "Omni-Path")
+echo "Found $opa_count Omni-Path adapters."
 if [ "$opa_count" -gt 0 ]; then
     ((ib_count=opa_count)) || true
 fi
 
+last_pci_bus=''
 while IFS= read -r line; do
+    pci_bus="${line%.*}"
+    if [ "$pci_bus" == "$last_pci_bus" ]; then
+        # We only use one interface on a dual interface HBA
+        # Fortunately lspci appears to group them together
+        continue
+    fi
+    last_pci_bus="$pci_bus"
     mlnx_type="${line##*ConnectX-}"
     mlnx_type="${mlnx_type%]*}"
     if [ "$mlnx_type" -ge 6 ]; then
```

Comment:
mlnx_type  is 5+ for OFED drivers either NDR or HDR.

Reply:
Can investigate in the future.